src/components/editor/property/ResourceList.jsx

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React, { useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { getTemplateSearchResults } from 'sinopiaSearch'
+import shortid from 'shortid'
+import { newResource } from 'actionCreators/resources'
+import { findAuthorityConfigs } from 'utilities/authorityConfig'
+import { getPropertyTemplate } from 'selectors/resourceSelectors'
+
+export const newResourceErrorKey = 'newresource'
```

Comment:
This error is not surfaced to users. Needs an `<Alerts>` added to ResourceTemplate.jsx referencing this error key.

Reply:
Fixed in latest commit.